src/mlpack/methods/ann/layer/convolution_impl.hpp

```diff
@@ -178,14 +178,15 @@ void Convolution<
 >::Forward(const arma::Mat<eT>& input, arma::Mat<eT>& output)
 {
   batchSize = input.n_cols;
-  arma::cube inputTemp(const_cast<arma::Mat<eT>&>(input).memptr(),
-      inputWidth, inputHeight, inSize * batchSize, false, false);
+  arma::cube inputTemp = arma::cube(const_cast<arma::Mat<eT>&>(input).memptr(),
+  inputWidth, inputHeight, inSize * batchSize, false, false);
```

Comment:
Any particular reason why we need to make this change? I might be missing something. Thanks.

Reply:
Sorry about that, its a merge issue. I'll revert that part